mitodl/edx-username-changer
# edx username changer plugin base

### Related Ticket
https://github.com/mitodl/edx-platform/issues/237

### What this PR does
It uses [edx-username-updater](https://github.com/mitodl/edx-username-updater) scripts to create a plugin for Open edX to be used for changing username in edx environments.

### How to test
Follow the brief configuration process defined under README.md of this PR. After configuration, create some discussion-forum posts with an account and change that account's username and see if the changes are reflected in posts and all other possible occurrences of the username or not.
